src/main/java/nus/climods/ui/common/HelpWindow.java

```diff
@@ -67,6 +82,31 @@ public void show() {
         logger.fine("Showing help page about the application.");
         getRoot().show();
         getRoot().centerOnScreen();
```

Comment:
Should have getRoot().setMaximised(true) here so that the help window pops up in full screen. Currently it shows up on mine with the top cropped out due to the pref-height.

Reply:
I kind of intended it to be not full screen, so that the user can see that they have not left the application.

Reply:
new idea:
We can get the screen bounds using:
`    Rectangle2D screenBounds = Screen.getPrimary().getBounds();`
Then we can just scale the `Help` Window size using this so that it does not get cropped out

Reply:
Thanks for your suggestion